lib/tasks/compare/ecf_users_and_induction_records.rake

```diff
@@ -10,32 +10,53 @@ namespace :compare do
       induction_records = Api::V1::ECF::InductionRecordsQuery.new.all
       serialized_induction_records = Api::V1::ECFInductionRecordSerializer.new(induction_records).serializable_hash
 
-      blue_data = serialized_users[:data].map { |u| u[:id] }
-      green_data = serialized_induction_records[:data].map { |u| u[:id] }
+      ids_from_serialized_users = serialized_users[:data].map { |u| u[:id] }
+      ids_from_serialized_induction_records = serialized_induction_records[:data].map { |u| u[:id] }
 
-      duplicate_records = green_data.tally.filter { |_, v| v > 1 } # assuming we're counting the number of duplications rather than duplicated records
-      same_records      = (blue_data & green_data)
-      new_records       = green_data - same_records
-      lost_records      = blue_data - same_records
+      duplicate_records = ids_from_serialized_induction_records.tally.filter { |_, v| v > 1 } # assuming we're counting the number of duplications rather than duplicated records
+      same_records      = (ids_from_serialized_users & ids_from_serialized_induction_records)
+      ir_but_not_users  = ids_from_serialized_induction_records - ids_from_serialized_users
+      users_but_not_ir  = ids_from_serialized_users - ids_from_serialized_induction_records
 
-      if new_records.count.zero? && lost_records.count.zero?
+      if ids_from_serialized_users.count.zero? && ids_from_serialized_induction_records.count.zero?
```

Comment:
Should this check `ir_but_not_users` and `users_but_not_ir` instead? As we don't want to return an error if we have records in the initial serialised responses?